snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/SnomedAxiomSemanticIT.java

```diff
@@ -0,0 +1,99 @@
+package dev.ikm.tinkar.snomedct.integration;
+
+import dev.ikm.tinkar.common.service.CachingService;
+import dev.ikm.tinkar.common.service.PrimitiveData;
+import dev.ikm.tinkar.common.service.ServiceKeys;
+import dev.ikm.tinkar.common.service.ServiceProperties;
+import dev.ikm.tinkar.common.util.uuid.UuidT5Generator;
+import dev.ikm.tinkar.coordinate.stamp.StampCoordinateRecord;
+import dev.ikm.tinkar.coordinate.stamp.StampPositionRecord;
+import dev.ikm.tinkar.coordinate.stamp.StateSet;
+import dev.ikm.tinkar.coordinate.stamp.calculator.Latest;
+import dev.ikm.tinkar.coordinate.stamp.calculator.StampCalculator;
+import dev.ikm.tinkar.entity.EntityService;
+import dev.ikm.tinkar.entity.PatternEntityVersion;
+import dev.ikm.tinkar.entity.SemanticRecord;
+import dev.ikm.tinkar.entity.SemanticVersionRecord;
+import dev.ikm.tinkar.terms.TinkarTerm;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class SnomedAxiomSemanticIT {
+
+    @BeforeAll
+    public static void setup() {
+        CachingService.clearAll();
+        File datastore = new File(System.getProperty("user.home") + "/Solor/generated-data"); //Note. Dataset needed to be generated within repo, with command 'mvn clean install'
+        ServiceProperties.set(ServiceKeys.DATA_STORE_ROOT, datastore);
+        PrimitiveData.selectControllerByName("Open SpinedArrayStore");
+        PrimitiveData.start();
+    }
+
+    @AfterAll
+    public static void shutdown() {
+        PrimitiveData.stop();
+    }
+
+    /**
+     * Test Snomed Axiom Semantics.
+     *
+     * @result Reads content from file and validates Snomed Axiom of Semantics by calling private method assertSnomedAxioms().
+     */
+    @Test
+    public void testSnomedAxiomSemantics() throws IOException {
```

Comment:
Similarly, you can just call this "testAxiomSemantics". The "Snomed" is redundant.

Reply:
Method renamed.